saleor/graphql/views.py

```diff
@@ -80,7 +75,7 @@ def __init__(
     def dispatch(self, request, *args, **kwargs):
         # Handle options method the GraphQlView restricts it.
         if request.method == "GET":
-            if settings.DEBUG:
+            if settings.PLAYGROUND_ENABLED:
```

Comment:
We agreed with @patrys and @cmiacz that Playground can be served always by default, so I wouldn't introduce another variable for that. We can also accept this change but it will be removed soon anyway.

Reply:
Then instead of `PLAYGROUND_ENABLED = getbool(..., DEBUG)` we could set to `True` by default? I prefer the idea of letting the user decide whether or not to enable it.

Reply:
Yes, if we're adding it then it should be set to `True`. There is no much difference whether you enable or disable it, anyone can access your API from their standalone playground or any other client.